cli/lsp/tsc.rs

```diff
@@ -4514,29 +4507,41 @@ fn op_script_names(state: &mut OpState) -> ScriptNames {
   let scopes_with_node_specifier =
     state.state_snapshot.documents.scopes_with_node_specifier();
   if scopes_with_node_specifier.contains(&None) {
-    result
-      .unscoped
-      .insert("asset:///node_types.d.ts".to_string());
+    let req_ref =
+      deno_semver::npm::NpmPackageReqReference::from_str("npm:@types/node")
```

Comment:
Why `npm:@types/node` instead of the asset file?

Reply:
It was so it could be turned into a file URL here and have proper scope attribution, but I passed the scope to `Documents::resolve()` instead to use as a fallback (when resolving `npm:@types/node` from `asset:///node_types.d.ts`).